app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -62,6 +58,10 @@ data class OverdueAppointment(
 
           (
             CASE
+              WHEN BloodSugar.reading_type = "fasting" AND CAST (BloodSugar.reading_value AS REAL) >= 200 THEN 1
+              WHEN BloodSugar.reading_type = "random" AND CAST (BloodSugar.reading_value AS REAL) >= 300 THEN 1
+              WHEN BloodSugar.reading_type = "post_prandial" AND CAST (BloodSugar.reading_value AS REAL) >= 300 THEN 1
```

Comment:
```suggestion
              WHEN (BloodSugar.reading_type = "random" OR BloodSugar.reading_type == "post_prandial") AND CAST (BloodSugar.reading_value AS REAL) >= 300 THEN 1
```

We can maybe replace both cases with this, wdyt?

Reply:
To maintain consistency, let's keep this separate?